sdbus_async/bluez/agent_api.py

```diff
@@ -54,3 +54,28 @@ async def request_default_agent(
         agent_path: str,
     ) -> None:
         raise NotImplementedError
+
+
+class AgentInterfaceAsync(
+    DbusInterfaceCommonAsync,
+    interface_name='org.bluez.Agent1',
+):
+    @dbus_method_async()
+    async def cancel(self) -> None:
+        raise NotImplementedError
+
+    @dbus_method_async()
+    async def release(self):
```

Comment:
The `cancel` now has return types but the other methods still don't have them.

Sorry I should have pointed out that multiple methods are missing the return type hints.